test/check-application

```diff
@@ -58,6 +57,60 @@ class TestApplication(testlib.MachineCase):
         b.wait_present("#containers-containers")
         b.wait_in_text("#containers-containers", "No running containers")
 
+        # delete container busybox:latest from container listing toggle
+        b.click("#containers-containers-filter button")
+        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='all'] a")
+        b.wait_visible("#containers-containers")
+        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
```

Comment:
Please adjust the comment. This doesn't actually delete anything, just selects the display mode and a container details tab (.. it seems?)

Reply:
Yes, wrong comment. It just opened the collapsed table before clicking the button inside the table.